Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -1808,3 +1808,70 @@ def get_demisto_version():
         return demisto.demistoVersion()
     else:
         raise AttributeError('demistoVersion attribute not found.')
+
+
+def parse_date_string(date_string, date_format='%Y-%m-%dT%H:%M:%S'):
+    """
+        Parses the date_string function to the corresponding datetime object.
+
```

Comment:
lets add comment here that suggest to use better functions in datetime-Python3 or dateutil when possible.